test/topology_custom/test_repair.py

```diff
@@ -67,3 +70,77 @@ async def test_enable_compacting_data_for_streaming_and_repair_live_update(manag
     # After the update to the config above, the next repair should pick up the updated value.
     await manager.api.repair(node1.ip_addr, "ks", "tbl")
     assert (await get_injection_params(manager, node1.ip_addr, "maybe_compact_for_streaming"))["compaction_enabled"] == "true"
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_tombstone_gc_for_streaming_and_repair_compaction(manager):
+    """
+    Check that:
+    * enable_tombstone_gc_for_streaming_and_repair_compaction=1 works as expected
+    * enable_tombstone_gc_for_streaming_and_repair_compaction=0 works as expected
+    * enable_tombstone_gc_for_streaming_and_repair_compaction is live-update
+    """
+    cmdline = [
+            "--enable-compacting-data-for-streaming-and-repair", "1",
+            "--enable-tombstone-gc-for-streaming-and-repair-compaction", "1",
+            "--enable-cache", "0",
+            "--hinted-handoff-enabled", "0",
+            "--smp", "1",
+            "--logger-log-level", "api=trace:database=trace"]
```

Comment:
Are all these extra parameters (except the hinted-handoff) relevant to this test?